src/service/poc_lora.proto

```diff
@@ -167,8 +167,14 @@ message lora_poc_v1 {
   repeated lora_verified_witness_report_v1 unselected_witnesses = 4;
 }
 
+enum reward_type {
+  gateway = 0;
+  operational = 1;
+  oracles = 2;
```

Comment:
Pick plural or singular.. either `gateways` and `oracles`  or `gateway` and `oracle`. Since this talks about a single reward I suggest singular

Reply:
fixed, all now singular